src/flatbuffer_types.rs

```diff
@@ -314,3 +330,24 @@ impl From<CompressionFormat> for Option<CompressionCodec> {
         }
     }
 }
+
+impl From<RowAttribute> for SstRowAttribute {
+    fn from(value: RowAttribute) -> Self {
+        match value {
+            RowAttribute::Flags => SstRowAttribute::Flags,
+        }
+    }
+}
+
+#[allow(clippy::panic)]
+impl From<SstRowAttribute> for RowAttribute {
+    fn from(value: SstRowAttribute) -> Self {
+        match value {
+            SstRowAttribute::Flags => RowAttribute::Flags,
+            _ => panic!(
+                "Attempted to read SST written with unknown SstRowAttribute. Are you \
+            running an older version of SlateDB than was used to write the SST?"
+            ),
```

Comment:
I thought panicking would be appropriate here since there isn't really anything the user can do to recover from this exception outside of upgrade their SlateDB version and piping through a `Result` would be a pain. Can be done though if people want me to make that change!

Reply:
I'm fine with panic for now. As we layer on more attributes, we might want to address it. But for now, it's fine.